Source/core/SocketPort.cpp

```diff
@@ -1132,7 +1138,11 @@ namespace Core {
         socklen_t size = sizeof(address);
         SOCKET result;
 
+        #ifdef __LINUX__
+        if ((result = ::accept4(m_Socket, (struct sockaddr*)&address, &size, SOCK_CLOEXEC)) != SOCKET_ERROR) {
+        #else
```

Comment:
Could not find the accept4 in the windows documentation that quickly, but suggest we remove the preprocessor stuff anyway. I will test it on windows and if I run into an issue I will just create a local accept4 method in this compilation unit that will push the SOCK_CLOEXEC option onto the socket. I am afraid if I am not running into a compilation issue, we forget about this change.